openedx/core/djangoapps/kafka_consumer/docs/0001_kafka_consumer_command.rst

```diff
@@ -0,0 +1,39 @@
+Managing Kafka Consumers
+--------------
+
+Status
+======
+
+In Progress
+
+Context
+=======
+As outlined in the upcoming OEP-52, we have elected to go with Apache Kafka as
+our event bus implementation. The standard pattern for consuming events with
+Kafka is to poll in a loop and process messages as they come in. According to
+the Confluent team, it is a best practice to limit each consumer to a single
+topic.::
+    
+    consumer.subscribe(["topic"])
+    while True:
+        message = consumer.poll()
+        ## process message
+
+This ``while True`` loop means whatever is running this consumer will run infinitely and block whatever thread runs it from doing anything else. Thus, we can’t just run this code as part of our regular Django web server. 
+
+Decision
+========
+We will use Kubernetes to manage containers whose sole purpose is to run a management command, which in turn will run a polling loop against the specified topic.
+
+Options Considered
+==================
+
+#. Use celery tasks to consume a set number of messages at a time use our existing celery structure
+    * This would likely require us to use the celery-beat library to run the celery tasks on a constant basis. Celery Beat has known            performance issues and would likely not scale well as the number of topics increased.
+    
+#. Create a new ASG of EC2 instances dedicated to running a consumer management command, similar to how we create instances dedicated to running celery workers
```

Comment:
I don't want you to have to over-invest in this doc, and it is a decision doc that needs to be useful to the community.

1. I'd recommend removing anything about celery here, and letting the first section describe "Why not Celery?". 
2. If you feel strongly that we should include an option of piggy-backing compute on the existing celery workers, you could make that its own section, but if it were me, I would drop it entirely.  If you want, maybe a short and sweet bullet about how it wouldn't enable proper consumer scaling, and it would fight for compute resources with the existing celery workers.
3. I think your second bullet covers nicely "Why not ASGs and EC2s". Because Kubernetes is our current model for this. Again, review and update any "we" language here and above, and try to make it relevant to someone in the wider community.